lib/SparkPost/Transmission.php

```diff
@@ -89,6 +89,11 @@ class Transmission extends APIResource {
    * @return array API repsonse represented as key-value pairs
    */
   public function send( $transmissionConfig ) {
+    if(isset($transmissionConfig["startTime"]) && $transmissionConfig["startTime"] instanceof \DateTime)
+      {
+        $transmissionConfig["startTime"] = $transmissionConfig["startTime"]->format(\DateTime::ATOM);
```

Comment:
@beardyman I've checked the API docs and there are no other parameters implemented in the php-sparkpost library that accept a `DateTime` other than the `Transmission` startDate.  I don't mind doing it, but the time to refactor it up to the base class would be if & when it needs to be used elsewhere wouldn't it?  

If it goes in the base class then presumably all parameters would be `instanceof` checked to see if they are `DateTime` objects?